build_test/lib/src/test_builder.dart

```diff
@@ -39,7 +38,7 @@ AssetId _passThrough(AssetId id) => id;
 /// association to a package pass [mapAssetIds] to translate from the logical
 /// location to the actual written location.
 void checkOutputs(
-    Map<String, /*List<int>|String|Matcher<List<int>>*/ dynamic> outputs,
+    Map<String, /*List<int>|String|Matcher<List<int>>*/ dynamic>? outputs,
```

Comment:
Generally we are switching over to `Object` or `Object?` from `dynamic` where we notice it. Not a big thing but does help prevent accidental dynamic calls and forces explicit casts instead.

```suggestion
    Map<String, /*List<int>|String|Matcher<List<int>>*/ Object>? outputs,
```